### DISCUSSION TITLE
align three columns with layout = bsp

### DISCUSSION THREAD

**Discussion by molleweide** [asmvik/yabai#1093] [Q&A] [answered]
Hi guys,

Is it possible to achieve a three column layout with bsp? I have not been able to do this so far.

**Comment by Liquidmantis** [comment, marked as answer]
For me sometimes it just happens.  Off the top of my head I don't have a good answer, but you can use the location-awareness of drag-and-dropping.
For example if you have three windows in the default 1/2, 1/4, 1/4 layout (sorry, I ain't got time to hack out an ASCII example right now...), then drag one of the 1/4 windows to the left or right edge of the other and you should end up with three columns.  You can also use this to go back to the default BSP layout, dragging one of the columnar windows on top of the top or bottom region of one of the others.

**Reply by molleweide** [reply]
Thanks for replying!!
I'll try your suggestion.
The main take away for me is that one might need to use the mouse in order to achieve it.

**Reply by asmvik** [reply]
https://github.com/koekeishiya/yabai/issues/157

**Reply by molleweide** [reply]
Thank you very much!
